terraform/provisioning/common.bash

```diff
@@ -45,7 +45,7 @@ common::caddy_run() {
     --network host \
     --volume=/etc/caddy/Caddyfile:/etc/Caddyfile:ro \
     --volume=/var/lib/caddy/dotcaddy:/root/.caddy:rw,Z \
-    quay.io/schu/caddy:0.11.5
+    quay.io/kinvolk/caddy:master
```

Comment:
Could we stay on a tagged version, so that container-escape-bounty's dependencies don't change implicitly?

Reply:
tagged `1.0.0` based on the caddy version!